apps/ccd/base/kustomization.yaml

```diff
@@ -3,4 +3,5 @@ kind: Kustomization
 resources:
   - ../../base
   - ../identity/identity.yaml
+  - ../logstash
```

Comment:
Is there a logstash folder in that location?

Reply:
yeah, i meant this https://github.com/hmcts/cnp-flux-config/blob/aa81b3997c9ba8980605d8c2902de71205212a43/apps/ccd/logstash/kustomization.yaml

Reply:
It's a bit of special setup :) It's in ccd namespace, but managed seperately.